docs/integrations/dbot.md

```diff
@@ -35,6 +35,14 @@ Dbot uses an integer to represent the reputation of an indicator.
 | 2 | Suspicious |
 | 3 | Bad |
 
+## Unknown
+Unknown score can be interpeted in two ways: 
+
+1. The vendor returns an "Unknown" score for the indicator.
+2. The vendor returns nothing on the indicator.
```

Comment:
I think the descriptions here are a bit incorrect.
1. The indicator do not exists in the product/integration.
2. The indicators in the product have no score/vector (only data)

Reply:
about 1 i tend to agree it is better. 
about 2 - there is a case that the vendor actually returns "Unknown" for the indicator

so maybe: 
1. The vendor returns an "Unknown" score for the indicator.
2. The indicator do not exists in the product/integration.
3. The indicators in the product have no score/vector (only data)

WDYT